functions/controllers/processCalendlyEventInvitee.js

```diff
@@ -45,18 +46,8 @@ async function handlePostRequest (req, res) {
   await addTagsToMailchimpSubscriber({
     listId: MAILCHIMP_DEFAULT_LIST_ID,
     subscriberId: subscriber.id,
-    tags: getMailchimpTagsFromCalendlyInvitee(invitee)
+    tags: getMailchimpTags(invitee)
```

Comment:
If you're gonna move this function to a general service and change its name, then you also need to change its signature. It shouldn't receive the Calendly invitee directly. Now, it should only receive the `source`:
```suggestion
    tags: getMailchimpTags(invitee.tracking.utm_campaign)
```